package.json

```diff
@@ -55,16 +56,15 @@
     "@types/react": "17.0.37",
     "@types/react-dom": "17.0.11",
     "@types/styled-components": "5.1.18",
-    "@typescript-eslint/eslint-plugin": "4.33.0",
-    "@typescript-eslint/parser": "4.33.0",
+    "@typescript-eslint/eslint-plugin": "5.7.0",
+    "@typescript-eslint/parser": "5.7.0",
     "@zendeskgarden/css-bedrock": "9.0.0",
-    "@zendeskgarden/eslint-config": "21.0.0",
+    "@zendeskgarden/eslint-config": "26.0.0",
     "@zendeskgarden/scripts": "1.4.0",
     "@zendeskgarden/stylelint-config": "16.0.1",
     "@zendeskgarden/svg-icons": "6.30.2",
     "acorn-jsx": "5.3.2",
     "babel-core": "7.0.0-bridge.0",
-    "babel-eslint": "10.1.0",
```

Comment:
[babel-eslint](https://www.npmjs.com/package/babel-eslint) is deprecated and replaced with now `@babel/eslint-parser`.